### DISCUSSION TITLE
How to implement folders in the media library

### DISCUSSION THREAD

**Discussion by stemcc** [strapi/strapi#6814] [Legacy Topics]
Right now the filtering options in the media libary are pretty basic. It would be extremely useful if we could use folders inside the media library to seperate large groups of media that belong together. I've not been able to find anything related to this, however that might be because I'm relatively new to working with Strapi.

**Comment by lauriejim** [comment]
Thank you for this request. For sure I think it will be in the v2 of the plugin.
I shared your message with the team to discuss about that.

**Comment by Aurelsicoko** [comment]
We decided to not release the folders in the first iteration of the Media Library. Until we work on the next version, I recommend you to have strong naming conventions and use the "Note" field as a tag system to easily group and find assets using the filters.
